### PR TITLE
Feat/add owallet

### PR DESCRIPTION
**Pull request by toandq2009**
[x] Add Owallet extension adapter
[x] Update README , include link to Owallet extension
[x] Passed lint  & test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,19 +106,18 @@ lerna publish
 
 Checkout these related projects:
 
-* [@cosmology/telescope](https://github.com/cosmology-tech/telescope) Your Frontend Companion for Building with TypeScript with Cosmos SDK Modules.
```
Comment (tucq88) [inline README.md:106]: Unnecessary change

**Review by Jaycat66 — APPROVED** [review]

**Comment by Jaycat66** [comment]
[asset-manifest.json](https://github.com/cosmology-tech/cosmos-kit/files/14468669/asset-manifest.json)

**Comment by Jaycat66** [comment]
[zenledger-20230929-163048.csv](https://github.com/cosmology-tech/cosmos-kit/files/14468677/zenledger-20230929-163048.csv)

**Review by Jaycat66 — CHANGES_REQUESTED** [review]
[zenledger-20230929-163048.csv](https://github.com/cosmology-tech/cosmos-kit/files/14468682/zenledger-20230929-163048.csv)
[README.md](https://github.com/cosmology-tech/cosmos-kit/files/14468683/README.md)
[asset-manifest.json](https://github.com/cosmology-tech/cosmos-kit/files/14468687/asset-manifest.json)

**Comment by twhy** [comment]
@toandq2009 Hi, I can't merge this because this branch has conflicts with main branch. Can you try merge main and try resolve the conflicts first ?

**Comment by toandq2009** [comment]
> @toandq2009 Hi, I can't merge this because this branch has conflicts with main branch. Can you try merge main and try resolve the conflicts first ?

Thank you, twhy. I have resolved the conflicts on this branch. Now you can merge my branch.

**Pull request merged**